Constellation/Node/Main.hs

```diff
@@ -86,7 +87,10 @@ run cfg@Config{..} = do
             }
     ast <- mustLoadPublicKeys cfgAlwaysSendTo
     logf' "Initializing storage {}" [cfgStorage]
-    storage <- berkeleyDbStorage cfgStorage
+    storage <- case break (== ':') cfgStorage of
+        ("bdb", ':':s) -> berkeleyDbStorage s
+        ("dir", ':':s) -> directoryStorage s
+        _              -> berkeleyDbStorage cfgStorage  -- Default
     -- storage <- memoryStorage
```

Comment:
Yeah was leaning the same way. Will open another PR soon, including some benchmarking code as well...